src/portals/blockchain-frontend/apps/blockchain-frontend/app/(authenticated)/trainees/page.tsx

```diff
@@ -31,9 +27,10 @@ export function CartPageTrainees(props: CartPageTraineesProps) {
   },[])
 
   const getData =() =>{
-    axios.get('http://localhost:7250/api/Trainee')
+    axios.get('http://localhost:7250/api/TraineeGetAPI?pageSize=10&PageNumber=1')
```

Comment:
check if we can use async await instead of then and catch. also put some loaders

Reply:
Done